Gruntfile.js

```diff
@@ -1,10 +1,24 @@
+'use strict';
+
 module.exports = function(grunt) {
-  'use strict';
+  require('load-grunt-tasks')(grunt);
   grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
     eslint: {
       src: ["lib/**/*.js"]
+    },
+    shell: {
+      unit: {
+        options: {
+          stdout: true,
+          stderr: true
+        },
+        command: 'env NODE_PATH=. ./node_modules/.bin/mocha -A -u exports --recursive **/*-spec.js'
+      }
     }
```

Comment:
we moved to [grunt-mocha-test](https://github.com/pghalliday/grunt-mocha-test) ref. #12